File: Estrutura_Condicional/C++/Ex1050/main.cpp
#include <iostream>
using namespace std;

int main() {
    int D;
    cin >> D;

    if (D == 61)
        cout << "Brasilia" << endl;
    else if (D == 71)
        cout << "Salvador" << endl;
    else if (D == 11)
        cout << "Sao Paulo" << endl;
    else if (D == 21)
        cout << "Rio de Janeiro" << endl;
    else if (D == 32)
        cout << "Juiz de Fora" << endl;
    else if (D == 19)
        cout << "Campinas" << endl;
    else if (D == 27)
        cout << "Vitoria" << endl;
    else if (D == 31)
        cout << "Belo Horizonte" << endl;
    else
        cout << "DDD nao cadastrado" << endl;

    return 0;
}

File: Estrutura_Condicional/C#/Ex1050/Main.cs
using System;

namespace Ex1050
{
    class Program
    {
        static void Main(string[] args)
        {
            int D;
            D = int.Parse(Console.ReadLine());

            if (D == 61)
            {
                Console.WriteLine("Brasilia");
            }
            else if (D == 71)
            {
                Console.WriteLine("Salvador");
            }
            else if (D == 11)
            {
                Console.WriteLine("Sao Paulo");
            }
            else if (D == 21)
            {
                Console.WriteLine("Rio de Janeiro");
            }
            else if (D == 32)
            {
                Console.WriteLine("Juiz de Fora");
            }
            else if (D == 19)
            {
                Console.WriteLine("Campinas");
            }
            else if (D == 27)
            {
                Console.WriteLine("Vitoria");
            }
            else if (D == 31)
            {
                Console.WriteLine("Belo Horizonte");
            }
            else
            {
                Console.WriteLine("DDD nao cadastrado");
            }
        }
    }
}
